src/components/confirmOrderComponents/OrderSummary.js

```diff
@@ -18,20 +21,19 @@ function OrderSummary({ buyOrSell, orderType, amountSelected, newPortfolioBalanc
                             </tr>
                             <tr>
                                 <td>Quantity</td>
-                                <td>{parseFloat(qty).toFixed(2)} stocks</td>
+                                <td>{parseFloat(qty).toLocaleString('en-US', {minimumFractionDigits: 2, maximumFractionDigits: 2})} stocks</td>
                             </tr>
                             <tr>
                                 <td>New Portfolio Balance</td>
-                                <td>${parseFloat(newPortfolioBalance).toFixed(2)}</td>
+                                <td>${parseFloat(newPortfolioBalance).toLocaleString('en-US', {minimumFractionDigits: 2, maximumFractionDigits: 2})}</td>
                             </tr>
                             <tr>
                                 <td>Total Cost</td>
-                                <td>${parseFloat(amountSelected).toFixed(2).toString()}</td>
+                                <td>${parseFloat(amountSelected).toLocaleString('en-US', {minimumFractionDigits: 2, maximumFractionDigits: 2})}</td>
```

Comment:
Just wondering why this is EN-US and not EN-IR or whatever the Irish version is?

Reply:
Just went with the US one because everything is in dollars in the game.